dcos_launch/config.py

```diff
@@ -196,16 +204,17 @@ def get_validated_config(user_config: dict, config_dir: str) -> dict:
     'key_helper': {
         'type': 'boolean',
         'default': False},
-    'zen_helper': {
-        'type': 'boolean',
-        'default': False},
     'tags': {
         'type': 'dict',
-        'required': False}}
+        'required': False}
+}
 
 
 TEMPLATE_DEPLOY_COMMON_SCHEMA = {
     # platform MUST be equal to provider when using templates
+    'deployment_name': {
```

Comment:
deployment_name is inherited, no reason to repeat: https://github.com/dcos/dcos-launch/blob/master/dcos_launch/config.py#L165

Reply:
@mellenburg No longer inherited in this branch. I removed it from the COMMON_SCHEMA because it will not be used by terraform. Terraform launches resources, but they do not seem to be grouped under a same deployment name. Also resource names are not currently configurable in terraform.

Reply:
Is the "owner" parameter the only way things get customized names? If so then this is making more sense to me

Reply:
yeah it's the only way. Resources names will be for example: $owner-3928-external-dcos-public-agent-address